concourse/util.py

```diff
@@ -173,3 +177,51 @@ def url_filter(url):
     )
     if removed > 0:
         info('removed {c} outdated hook(s)'.format(c=removed))
+
+
+def sync_org_webhooks(
+    whd_deployment_cfg: WebhookDispatcherDeploymentConfig,
+):
+    cfg_factory = ctx().cfg_factory()
+
+    whd_cfg_name = whd_deployment_cfg.webhook_dispatcher_config_name()
+    whd_cfg = cfg_factory.webhook_dispatcher(whd_cfg_name)
+
+    concourse_cfg_names = whd_cfg.concourse_config_names()
+    concourse_cfgs = map(cfg_factory.concourse, concourse_cfg_names)
+
+    for concourse_cfg in concourse_cfgs:
+        job_mapping_set = cfg_factory.job_mapping(concourse_cfg.job_mapping_cfg_name())
+
+        for github_orgname, github_cfg_name in _enumerate_github_org_configs(job_mapping_set):
+            github_api = _create_github_api_object(
+                github_cfg=cfg_factory.github(github_cfg_name),
+            )
+
+            webhook_url = _build_webhook_url(
+                ingress_host_url=whd_deployment_cfg.ingress_host(),
+                whd_cfg_name=whd_cfg_name,
+            )
```

Comment:
so actually, my suggestions would be:
- add a `yield (github_api, github_org_name, webhook_url)` here
- rename this function (something like "enumerate_required_org_webhooks")
- call this function from a (then new) function "sync(_org?)_webhooks; iterate over the returned three-tuples in said function

Reply:
Done